commons/src/main/java/io/aiven/kafka/tiered/storage/config/validators/NonEmptyPassword.java

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package io.aiven.kafka.tieredstorage.storage.config;
+package io.aiven.kafka.tiered.storage.config.validators;
```

Comment:
is the change from `tieredstorage` to `tiered.storage` on purpose?

Reply:
Nope, fixed